modules/bindbackend/bindbackend2.cc

```diff
@@ -131,7 +131,8 @@ void BB2DomainInfo::setCtime()
   d_ctime = buf.st_ctime;
 }
 
-bool Bind2Backend::safeGetBBDomainInfo(int id, BB2DomainInfo* bbd)
+// NOLINTNEXTLINE(readability-identifier-length)
```

Comment:
I think I said it before in internal chat, I think this check is too much of a burden and we should get rid of it